src/rpc/error.rs

```diff
@@ -15,6 +16,13 @@ pub struct ServerError {
     inner: ErrorObjectOwned,
 }
 
+/// Custom error codes for the Forest node.
+pub enum ForestError {
+    /// This error indicates that the execution reverted while executing the message.
+    /// Code is taken from https://github.com/filecoin-project/lotus/blob/master/api/api_errors.go#L27
```

Comment:
Could you provide a stable link (with a specific lotus version) to prevent code changes under the hood?

Reply:
Done